examples/speaker_tasks/diarization/conf/neural_diarizer/sortformer_diarizer_hybrid_loss_4spk-v1.yaml

```diff
@@ -0,0 +1,218 @@
+# Sortformer Diarizer is an end-to-end speaker diarization model that is solely based on Transformer-encoder type of architecture.
+# Model name convention for Sortformer Diarizer: sortformer_diarizer_<FC_layer>_<TF_layer>_<loss_type>_loss.yaml
+# (Example) `sortformer_diarizer_FC18_TF18_hybrid_loss.yaml` has 18 layers for FastConformer and 18 layers of Transformer.
+# Sortformer Diarizer model checkpoint (.ckpt) and NeMo file (.nemo) contain Fast Conformer Encoder model (NEST Encoder) and the pre-trained NEST model is loaded along with the Transformer Encoder layers.
+# Example: a manifest line for training 
+# {"audio_filepath": "/path/to/audio01.wav", "offset": 390.83, "duration": 90.00, "text": "-", "num_speakers": 2, "rttm_filepath": "/path/to/audio01.rttm"}
+name: "SortFormerDiarizer"
+sample_rate: 16000
+num_workers: 18
+batch_size: 8
+
+model: 
+  pil_weight: 0.5
+  ats_weight: 0.5
+  num_workers: ${num_workers}
+  fc_d_model: 512
+  tf_d_model: 192
+  max_num_of_spks: 4 # Number of speakers per model. This is currently fixed at 4.
+  session_len_sec: 90
+
+  train_ds:
+    manifest_filepath: ???
+    sample_rate: ${sample_rate}
+    num_spks: ${model.max_num_of_spks}
+    session_len_sec: ${model.session_len_sec}
+    soft_label_thres: 0.5
+    soft_targets: False
```

Comment:
Maybe add some short explanation for `soft_label_thres` and `soft_targets`?

Reply:
Added some comments